### DISCUSSION TITLE
Theme light/dark mode

### DISCUSSION THREAD

**Discussion by nico-bachner** [slidevjs/slidev#85] [Q&A] [answered]
I'm currently working on a new theme for slidev, but I'm having difficulties wrapping my head around the light/dark mode situation. I see the theme switch is on the slide, so I assume it's using a class-based toggle (or something of the likes). 

My question is: how do you access the state of the switch?

**Comment by antfu** [comment, marked as answer]
```ts
import { isDark, toggleDark } from '@slidev/client/logic/dark'
```

You can see https://github.com/slidevjs/slidev/blob/7af7b6864b7d999a94e5eb0e1af9011b907470f4/demo/composable-vue/components/DarkToggle.vue

**Reply by nico-bachner** [reply]
Thanks Anthony! Will try again with that.
